docs/.vuepress/handsontable-manager/dependencies.js

```diff
@@ -2,23 +2,52 @@
 const isBrowser = (typeof window !== 'undefined');
 
 const formatVersion = version => (/^\d+\.\d+$/.test(version) ? version : 'latest');
-const getHotUrls = (version) => {
+
+/**
+ * Gets the local/remote package url for the required file type.
+ *
+ * @param {string} packageName The package name.
+ * @param {string} version The package version. If specified as 'next', the local urls will be returned.
+ * @param {'js'|'css'|string} fileSelection If set to `js` or `css`, it will return the urls of the default js/css
+ * files. If any other value is provided, it will be treated as a path the required file.
+ * @returns {string} Url to the required file.
+ */
+const getPackageUrls = (packageName, version, fileSelection) => {
+  const subDirs = {
+    handsontable: {
+      js: 'handsontable.full.min.js',
+      css: 'handsontable.full.min.css'
+    },
+    '@handsontable/react': {
+      js: 'react-handsontable.js'
```

Comment:
Why here is the non-minified version used when all others use min? 😄

Reply:
Good point